src/components/Button/SecondaryButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComponentProps } from 'react';
+import clsx from 'clsx';
+
+import { ChildrenFC } from '../../utils/type';
+import style from './SecondaryButton.module.scss';
+
+type ButtonProps = {
+  onClick?: () => void;
+  className?: string;
+  type?: 'button';
+} & ComponentProps<'button'>;
+
+export const SecondaryButton: ChildrenFC<ButtonProps> = ({
```

Comment:
I am not sure what is the difference, but if visual only (different colors). We should introduce some `variant` prop to the classic button and via clsx we can restyle it to whatever.

If you pass in the variant prop do in the props `({ foo, bar, variant = 'primary' })` so it always defaults to the primary variant if not provided

You would have `style.baseStyle` for the shared styles

`style.primary` for the classic variant
`style.secondary` for the secondary variant